assemble/bin/accumulo-cluster

```diff
@@ -116,19 +138,21 @@ function control_service() {
     if [[ $host == localhost || $host == "$(hostname -s)" || $host == "$(hostname -f)" || $host == $(get_ip) ]] ; then
       ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd"
     else
-      $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\"'"
+      $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"{$@:4}'"
```

Comment:
This looks incorrect.  There's no trailing escaped double-quote. Also, the quoting is extremely hard to reason about here... not sure it's a great idea to pass along additional arbitrary arguments to bash within ssh within bash when they could have their own quoting issues.